packages/flutter/test/widgets/platform_view_test.dart

```diff
@@ -669,4 +669,40 @@ void main() {
       ]),
     );
   });
+
+  testWidgets('Android view with eager gesture recognizer', (WidgetTester tester) async {
+    final int currentViewId = platformViewsRegistry.getNextPlatformViewId();
+    final FakePlatformViewsController viewsController = new FakePlatformViewsController(TargetPlatform.android);
+    viewsController.registerViewType('webview');
+    await tester.pumpWidget(
+      new Align(
+        alignment: Alignment.topLeft,
+        child: GestureDetector(
+          onVerticalDragStart: (DragStartDetails d) {},
+          child: SizedBox(
+            width: 200.0,
+            height: 100.0,
+            child: AndroidView(
+              viewType: 'webview',
+              gestureRecognizers: <OneSequenceGestureRecognizer> [ new EagerGestureRecognizer() ],
+              layoutDirection: TextDirection.ltr,
+            ),
+          ),
+        ),
+      ),
+    );
+
+    await tester.startGesture(const Offset(50.0, 50.0));
+
+    // Normally (without the eager gesture recognizer) after just the pointer down event
+    // no gesture arena member will claim the arena (so no motion events will be dispatched to
+    // the Android view). Here we assert that with the eager recognizer in the gesture team the
+    // pointer down event is immediately dispatched.
```

Comment:
might be worth adding the control test (that we don't dispatch the event if there's no eager thingy)

Reply:
This is cover by a test above ('Android view can lose gesture arenas')